code/src/Components/Result.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export const Result = ({ name, going, reward }) => {
+  const restartButtonClick = () => {
+    window.location.reload()
+  }
+  return (
+    <>
+      <div className="resultComponent">
+        <h2>You are doing great and you should be proud of yourself! 💪</h2>
+        <h3>This is the summary of your answers</h3>
+        <div className="results">
+          <p><span className="result-name">Your name:</span> {name}</p>
+          <p><span className="result-going">How is it going:</span> {going}</p>
+          <p><span className="result-reward">Reward:</span> {reward}</p>
```

Comment:
Clever to place these span elements inside the p elements, that sure allows for greater flexibility in styling the text, never thought of that!  Using semantic markup, with the span elements representing labels and the p elements representing paragraphs of text, it also makes the code more accessible to screen readers and other assistive technologies. Neat and impressive!